libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -378,6 +378,16 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("IO_DSHOT", 28, AP_BoardConfig, state.io_dshot, 0),
 #endif
+
+#if AP_CRASHDUMP_ACK_ENABLED
+    // @Param: CRSHDMP_ACK
+    // @DisplayName: CrashDump Acknowledgement Required
+    // @Description: Must have value "1" if a crashdump is present on the system, or prearm failure will be raised.  Do not set this parameter unless the risks of doing so are fully understood.  The presence of a crash dump means that the firmware currently installed has crashed with a hardfault or watchdog. The crash dump file gives diagnostic information which can help in finding the issue. Please check the ArduPilot documentation for more details.
```

Comment:
Arming check enable/disable when CrashDumps files are found on the SD Card.  CrashDumps are saved after  critical software failures resulting in the autopilot immediately rebooting.  If these are present the vehicle is likely unsafe to fly.  Please report to the ArduPilot support team.

Reply:
> Arming check enable/disable when CrashDumps files are found on the SD Card. CrashDumps are saved after critical software failures resulting in the autopilot immediately rebooting. If these are present the vehicle is likely unsafe to fly. Please report to the ArduPilot support team.

We do put crashdump data onto SD cards - but that's not the data this PR is checking.  CrashDump flashes it into the STM32 flash area.  We copy it from there onto the SD card on subsequent boots.

Reply:
I've adjusted the wording here.  It's turning into a bit of an essay.